hack/helmfile.yaml

```diff
@@ -118,13 +118,14 @@ releases:
     chart: '../.local/charts/nginx-ingress-services'
     values:
       - './helm_vars/nginx-ingress-services/values.yaml.gotmpl'
-      - './helm_vars/nginx-ingress-services/certificates-namespace1.yaml'
```

Comment:
`certificates-namespace1.yaml` and `certificates-namespace2.yaml` are still in `.gitignore`. I think they can be removed there, too.